### PR TITLE
Update tutorial on stdin and add tests

### PR DESCRIPTION
**Pull request by brunats**
**- What I did**
Updated tutorial change code via stdin and included tests.

**- How to verify it**
> Clone this pr with the command and build Ritchie
```bash
git fetch upstream pull/426/head:improvement/update-stdin-for-tutorial-command
git checkout improvement/update-stdin-for-tutorial-command
make build && sudo cp dist/linux/rit /usr/local/bin/
```
> And run tutorial via stdin
```bash
cat ~/.rit/tutorial.json # Check tutorial status
echo '{"tutorial": "disabled"}' > entry.json # Create file tutorial.json with json struct
rit tutorial --stdin < entry.json
cat ~/.rit/tutorial.json
```

**- Description for the changelog**

Updated tutorial change code via stdin and included tests.

### CODE REVIEW

File: pkg/cmd/tutorial.go [hunk 1]
```diff
@@ -55,17 +54,20 @@ func NewTutorialCmd(homePath string, il prompt.InputList, fs rtutorial.FindSette
...
-
 		obj := struct {
 			Tutorial string `json:"tutorial"`
 		}{}
 
-		err := stdin.ReadJson(os.Stdin, &obj)
+		err := stdin.ReadJson(cmd.InOrStdin(), &obj)
 		if err != nil {
 			return err
 		}
```
Comment (victor-schumacher) [inline pkg/cmd/tutorial.go:64]: What you think about:
```
if err := stdin.ReadJson(cmd.InOrStdin(), &obj); err != nil {
    return err 
}

Comment (brunasilvazup) [reply pkg/cmd/tutorial.go:64]: Very good, I will make the change.

File: pkg/cmd/tutorial.go [hunk 2]
```diff
@@ -55,17 +54,19 @@ func NewTutorialCmd(homePath string, il prompt.InputList, fs rtutorial.FindSette
...
 		}{}
 
-		err := stdin.ReadJson(os.Stdin, &obj)
-		if err != nil {
+		if err := stdin.ReadJson(cmd.InOrStdin(), &obj); err != nil {
 			return err
 		}
 
-		fmt.Println(obj)
+		_, err := o.Set(obj.Tutorial)
```
Comment (kaduartur) [inline pkg/cmd/tutorial.go:65]: Could you add a name for rtutorial.FindSetter interface?
Example: `o.tutorial.Set(obj.Tutorial)`

File: pkg/cmd/tutorial.go [hunk 3]
```diff
@@ -55,17 +54,19 @@ func NewTutorialCmd(homePath string, il prompt.InputList, fs rtutorial.FindSette
...
+		if err != nil {
```
Comment (kaduartur) [inline pkg/cmd/tutorial.go:66]: Could you simplify this error verification?

File: pkg/cmd/tutorial.go (hunk 2 above)
Comment (brunasilvazup) [reply pkg/cmd/tutorial.go:65]: Of course, I will change it.

File: pkg/cmd/tutorial.go (hunk 3 above)
Comment (brunasilvazup) [reply pkg/cmd/tutorial.go:66]: Yes, I will make it similar to the previous one.

File: pkg/cmd/tutorial_test.go [hunk 4]
```diff
@@ -33,3 +37,150 @@ func TestNewTutorialCmd(t *testing.T) {
...
+	if cmd == nil {
+		t.Errorf("NewTutorialCmd got %v", cmd)
+	}
+
+	if err := cmd.Execute(); err != nil {
+		t.Errorf("%s = %v, want %v", cmd.Use, err, nil)
+	}
+}
+
+func Test_tutorialCmd_runAnyEntry(t *testing.T) {
```
Comment (lucasdittrichzup) [inline pkg/cmd/tutorial_test.go:58]: the name of this test appears to be nonstandard. Is that correct?

**Review by kaduartur — DISMISSED** [review]
Good job! 🚀

File: pkg/cmd/tutorial_test.go (hunk 4 above)
Comment (brunasilvazup) [reply pkg/cmd/tutorial_test.go:58]: Yes, it is to be a genetic test for both types of entry (cmd and stdin) and then if necessary the specific tests per entry (nomenclature already known)

Comment (lucasdittrichzup) [reply pkg/cmd/tutorial_test.go:58]: the pattern I commented on is `camelcase` and you used `underline`

Comment (brunasilvazup) [reply pkg/cmd/tutorial_test.go]: Fix in https://github.com/ZupIT/ritchie-cli/pull/435/commits/628f9a9e3850d5280e009d7cd3f868970e3a1fbe

**Comment by lucasdittrichzup** [comment]
:rocket: :woman_astronaut:

**Pull request merged**